xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -6522,6 +6522,12 @@ TEST_P(ParameterizedFp8GemmRewriteTest, ScaledABScaledDMatrixBiasF8) {
   const char* hlo_text = R"(
     HloModule test
 
+    apply {
```

Comment:
Change the name to ScaledABScaledDMatrixBiasWithDAmaxF8 or add this as a new test.

Reply:
Done.